lib/tpm2_options.h

```diff
@@ -41,12 +41,19 @@
 
 typedef struct tpm2_options tpm2_options;
 
+#define tpm2_option_flags_init(x) { .all = x };
+
 typedef union tpm2_option_flags tpm2_option_flags;
 union tpm2_option_flags {
     struct {
+#define TPM2_OPTION_FLAG_VERBOSE       (1 << 0)
         UINT8 verbose : 1;
+#define TPM2_OPTION_FLAG_QUIET         (1 << 1)
         UINT8 quiet   : 1;
+#define TPM2_OPTION_FLAG_ENABLE_ERRATA (1 << 2)
         UINT8 enable_errata  : 1;
+#define TPM2_OPTION_NO_SAPI            (1 << 3)
+        UINT8 no_sapi : 1;
```

Comment:
@williamcroberts This was merged before I got time to review it :smiley:

It looks good to me, I just have one comment: could we define these constants together after the tpm2_option_flags_init() macro instead of interleaving them with the struct fields? I think that would make it much easier to read.

Reply:
Done, ill put them on a separate commit